vars/cloudFoundryDeploy.groovy

```diff
@@ -20,6 +20,25 @@ import groovy.transform.Field
          * @parentConfigKey cloudFoundry
          */
         'apiEndpoint',
+        /**
+         * Addition command line options for cf api command.
+         * No escaping/quoting is performed. Not recommanded for productive environments.
+         * @parentConfigKey cloudFoundry
+         */
+        'apiOpts',
+        /**
+         * Addition command line options for cf login command.
+         * No escaping/quoting is performed. Not recommanded for productive environments.
+         * @parentConfigKey cloudFoundry
+         */
+        'loginOpts',
+        /**
+         *
+         * Additional comand line options for cf deploy command.
+         * No escaping/quoting is performed. Not recommanded for productive environments.
+         * @parentConfigKey cloudFoundry
+         */
+        'deployOpts',
```

Comment:
The some deployments we already have some specific opts, e.g. mtaDeployParameters. 

Also about naming: There we call them parameters, here opts. Also not sure, if we want to use abbreviations in parameters.

Reply:
>Also not sure, if we want to use abbreviations in parameters.

The long versions are also OK for me. I used the short versions inspired by e.g. JAVA_OPTS.

I have some issues parsing

> The some deployments we already have some specific opts, e.g. mtaDeployParameters.

I guess the point is: mtaDeployParameters is the same like deployOpts. Seems to be some duplication, yes. How should we deal with that? Having additional options only for the deploy command seems to be not sufficient (e.g. `--skip-ssl-verification` needs to be provided for `api` and `login`.

What about changing the options for `api`, `login` to `mtaApiParameters`, and `mtaLoginParameters`? Does that sound reasonable from your point of view. I "yes" I can adjust accordingly.